plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryMetadata.java

```diff
@@ -292,6 +294,22 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
         return new ConnectorTableProperties();
     }
 
+    @Override
+    public void createSchema(ConnectorSession session, String schemaName, Map<String, Object> properties, TrinoPrincipal owner)
+    {
+        DatasetInfo datasetInfo = DatasetInfo.newBuilder(schemaName).build();
+        bigQueryClient.createSchema(datasetInfo);
+    }
+
+    @Override
+    public void dropSchema(ConnectorSession session, String schemaName)
+    {
+        String remoteSchemaName = bigQueryClient.toRemoteDataset(projectId, schemaName)
+                .map(RemoteDatabaseObject::getAnyRemoteName)
```

Comment:
Using `getAnyRemoteName` seems wrong here. We do not want to delete a random schema if two different match (module case sensitivity). It seem you should use `getOnlyRemoteName` here.